torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -170,6 +170,7 @@ AOTI_TORCH_EXPORT AOTITorchError aoti_torch_create_tensor_from_blob(
     const int64_t* strides_ptr,
     int64_t storage_offset,
     int32_t dtype,
+    int8_t layout,
```

Comment:
All the APIs in this file is supposed to be ABI-stable. If you need to add a parameter like this, you will have to create a v2 version of this function.